File: Bua.CodeRev.CompilerService/Bua.CodeRev.CompilerService.Core/Models/ExecutionRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Bua.CodeRev.CompilerService.Core.Models
{
    public class ExecutionRequest
    {
        public EntryPoint EntryPoint { get; set; }
        public string Code { get; set; }
    }
}

File: CodeRev/WebClient/src/app/code-editor/models/executionRequest.ts
import { ProgrammingLanguage } from 'src/app/review/models/programmingLanguage';
import { EntryPoint } from './entryPoint';

export class ExecutionRequest {
    constructor(
        public code: string,
        public entryPoint: EntryPoint,
        public programmingLanguage: ProgrammingLanguage) {}
}
